### DISCUSSION TITLE
Create file and open in adjacent panel

### DISCUSSION THREAD

**Discussion by aikruger** [SilentVoid13/Templater#897] [Help]
The open_new option with in the tp.file.create_new function opens up the new file replacing the note I am running the script from. Is it possible to open the new note in another panel without closing the note I am on?

**Comment by AB1908** [comment]
If this isn't implemented, you can do this via a workspace method to open in new leaf while keeping the `open` flag as false.

**Reply by aikruger** [reply]
sorry, newbie question what is a "workspace method"?

**Reply by AB1908** [reply]
`app.workspace` has a bunch of cool Obsidian APIs to manipulate your workspace.
